src/Compilers/VisualBasic/Test/Symbol/SymbolDisplay/SymbolDisplayTests.vb

```diff
@@ -6032,6 +6032,142 @@ end class"
                 actual:=displayParts)
         End Sub
 
+        <Theory, CombinatorialData>
+        Public Sub TestExtensionBlockCSharp_01(useMetadata As Boolean)
+            Dim text =
+<text>
+static class E
+{
+    extension(object o)
+    {
+        public void M() { }
+    }
+}
+</text>.Value
+
+            Dim format = New SymbolDisplayFormat(
+                                typeQualificationStyle:=SymbolDisplayTypeQualificationStyle.NameAndContainingTypesAndNamespaces,
+                                memberOptions:=SymbolDisplayMemberOptions.IncludeParameters Or
+                                               SymbolDisplayMemberOptions.IncludeModifiers Or
+                                               SymbolDisplayMemberOptions.IncludeAccessibility Or
+                                               SymbolDisplayMemberOptions.IncludeType Or
+                                               SymbolDisplayMemberOptions.IncludeContainingType,
+                                kindOptions:=SymbolDisplayKindOptions.IncludeMemberKeyword,
+                                parameterOptions:=SymbolDisplayParameterOptions.IncludeType Or
+                                                  SymbolDisplayParameterOptions.IncludeName Or
+                                                  SymbolDisplayParameterOptions.IncludeDefaultValue,
+                                miscellaneousOptions:=SymbolDisplayMiscellaneousOptions.UseSpecialTypes)
+
+            Dim parseOptions = CSharp.CSharpParseOptions.Default.WithLanguageVersion(CSharp.LanguageVersion.Preview)
+            Dim comp As Compilation
+            If useMetadata Then
+                Dim libComp = CreateCSharpCompilation("c", text, parseOptions:=parseOptions)
+                comp = CreateCompilation("", references:={libComp.EmitToImageReference()})
```

Comment:
>comp = CreateCompilation("", references:={libComp.EmitToImageReference()}) [](http://example.com/codeflow?start=16&length=74)

I think this isn't quite what I was asking to test. This creates VB compilation, right? I was asking: "testing behavior for C# symbols from metadata". I.e. test metadata symbols, but still created by C# compiler.